### PR TITLE
Switch calculation engine implementation

### PR DESCRIPTION
**Pull request by djbingham**
https://eaflood.atlassian.net/browse/FCEP-119

### CODE REVIEW

File: server/services/rulesEngineService.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    await this.doFullRun(eligibilityRules, parcels, passedFacts, successFunc)
+  },
+
+  doFullRun: async function (requestedRules, parcels, passedFacts, successFunc, returnFacts) {
+    try {
+      if (parcels.length !== 1) {
+        throw Error('rulesEngineService.doFullRun requires exactly 1 parcel')
+      }
+
+      const parcel = parcels[0]
```
Comment (jaucourt) [inline server/services/rulesEngineService.js]: You could just do `const [parcel] = parcels` here

File: test/routes/paymentCalculation.integration.test.js [hunk 2]
```diff
@@ -194,6 +194,7 @@ describe('POST /parcels/{parcelRef/actions/{actionId}/payment-calculation', () =
     })
   })
 
+  // Skip area rule tests because these rules are not yet implemented in new rules engine
```
Comment (jaucourt) [inline test/routes/paymentCalculation.integration.test.js]: We don't look to be skipping any tests - is this an old comment?

**Review by jaucourt — CHANGES_REQUESTED** [review]
Just a couple of little things and then we can merge. Nice work!

File: test/routes/paymentCalculation.integration.test.js (hunk 2 above)
Comment (djbingham) [reply test/routes/paymentCalculation.integration.test.js]: 😂 Yep, that's from before we updated the new rules engine to cover area rules.

File: server/services/rulesEngineService.js (hunk 1 above)
Comment (djbingham) [reply server/services/rulesEngineService.js]: Yep, done that.

File: test/routes/paymentCalculation.integration.test.js (hunk 2 above)
Comment (djbingham) [reply test/routes/paymentCalculation.integration.test.js]: And that's now removed. This is why I try so hard not to put comments in code. They always get forgotten about...

**Review by jaucourt — APPROVED** [review]

**Pull request merged**